src/Command/GeneratorCommand.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Emul\OpenApiClientGenerator\Command;
+
+use DI\Container;
+use Emul\OpenApiClientGenerator\Configuration\Configuration;
+use Emul\OpenApiClientGenerator\Configuration\Factory as ConfigFactory;
+use Emul\OpenApiClientGenerator\Generator\Factory as GeneratorFactory;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[AsCommand(
+    name: 'client-generator',
```

Comment:
Should be named as a verb so either `generate-client` or even better use symfony's command namespacing and call it `client:generate`